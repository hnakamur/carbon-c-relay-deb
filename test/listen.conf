# listen syntax incarnations
#
# 
# listen
#     type linemode [transport <gzip | lza4 | ssl <pemcert>>]
#         <<interface[:port] | port> proto <udp | tcp>> ...
#         </ptah/to/file proto unix> ...
#     ;

listen
	type linemode
		2013 proto tcp
		0.0.0.0:2013 proto udp
		/var/tmp/tstlisten.sock proto unix
	;

listen
	type linemode
		0.0.0.0:2014 proto tcp
	;

listen
	type linemode transport gzip
		0.0.0.0:2015 proto tcp
	;

listen
	type linemode transport ssl mycert.pem
		127.0.0.1:2913 proto tcp
	;
