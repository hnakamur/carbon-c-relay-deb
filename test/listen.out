listen
    type linemode
        2013 proto tcp
        0.0.0.0:2013 proto udp
        /var/tmp/tstlisten.sock proto unix
        0.0.0.0:2014 proto tcp
    type linemode transport gzip
        0.0.0.0:2015 proto tcp
    type linemode transport ssl mycert.pem
        127.0.0.1:2913 proto tcp
    ;

statistics
    submit every 60 seconds
    prefix with carbon.relays.test_hostname
    ;



